labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise02.scala

```diff
@@ -27,7 +32,7 @@ object ListManipulationExercise02 {
    * method above
    */
   def sumOfMany(l: List[Int]*): List[Int] = {
-    error("fix me")
+    l reduce sumOfTwo
```

Comment:
That was my solution too, but it fails for the empty list. A better solution would be to use `fold` and explicitly provide a base case of `Nil`.